provisioning/transport/http/src/HttpAuthStrategyTpm.cs

```diff
@@ -16,15 +18,24 @@ public HttpAuthStrategyTpm(SecurityProviderTpm security)
             _security = security;
         }
 
-        public override DeviceProvisioningServiceRuntimeClient CreateClient(Uri uri)
+        public override DeviceProvisioningServiceRuntimeClient CreateClient(Uri uri, IWebProxy proxy)
         {
             var serviceCredentials = new TpmCredentials();
             var tpmDelegatingHandler = new TpmDelegatingHandler(_security);
             var apiVersionDelegatingHandler = new ApiVersionDelegatingHandler();
 
+            HttpClientHandler httpClientHandler = new HttpClientHandler();
```

Comment:
This now seems common code - please extract in the common portion, leaving only TPM/X509 particular code in these classes.